Solidity Verifier

mehdi-defiesta opened the pull request:
## Description
<!-- Describe your changes in detail -->
Removed the useless debugs statements
Changes include solidity verifier updates only.
## Related Issue
<!-- Please link to the issue here -->
Closes #

## Type of Change
<!-- Please check the one that applies to this PR -->
- [ ] 🔥 Breaking Change
- [X] 🌟 New Feature
- [ ] 🐛 Bug Fix
- [X] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🔧 Code Refactoring
- [ ] 📈 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Testing
<!-- Please describe the tests you ran -->
- [X] Unit Tests
- [ ] Integration Tests
- [ ] Manual Tests

mehdi-defiesta commented:
@JehyukJang I removed the useless statements I added for debugging

JehyukJang commented on packages/backend/verify/verify-rust/src/lib.rs:
Still, there are debugging lines.

JehyukJang commented:
> @JehyukJang I removed the useless statements I added for debugging

Still, there is possibility of confliction. Please consider adding only "backend/verifier/solidity/" folder to your commit. No need to touch the readme files.

mehdi-defiesta commented:
@JehyukJang I’ve amended my commit to exclusively include changes under backend/verifier/solidity/ and removed modifications to README files, as you suggested. The PR is now rebased/updated with only the relevant changes.

Key adjustments made:
✅ Kept only changes in backend/verifier/solidity/.
✅ Removed all edits to README files.

Please let me know if further adjustments are needed.
